### ISSUE TITLE
vizion: false ignored in .json file after gracefulReload

### ISSUE THREAD

**Issue by sradu** [Unitech/pm2#2867]
I have a json file like:
```
{
  "apps" : [{
    "name"        : "api",
    "script"      : "app.js",
    "merge_logs"  : true,
    "exec_mode"   : "cluster",
    "instances"   : 2,
    "vizion"      : false,
  }]
}
```

During the initial run, using auditd, I can see that no git commands are being issued.

After a gracefulReload they are being run. I was expecting vizio to be disabled completely.

## Supporting information

```
PM2 version: 2.4.6
Node version: 7.9.0
Linux
```

**Comment by Unitech** [comment]
fixed, will be published on next release

**Comment by vmarchaud** [comment]
Published under PM2 `2.5`
